sdk/tables/data-tables/test/integration/tableclient.spec.ts

```diff
@@ -347,4 +355,80 @@ describe("TableClient", () => {
       assert.equal(result.floatingPointNumber, 3.14);
     });
   });
+
+  if (isNode || isLiveMode()) {
```

Comment:
There is a weird issue when executing these tests against browser and playback mode, live mode works fine, and test with any mode works for node.

I've filed https://github.com/Azure/azure-sdk-for-js/issues/11603 to investigate further. Meanwhile, I'm skipping the test if run in playback mode against the browser